src/test/java/org/primeframework/mvc/BrowserSessionTest.java

```diff
@@ -23,6 +23,12 @@
  * @author Brian Pontarelli
  */
 public class BrowserSessionTest extends PrimeBaseTest {
+  // Scenario:
```

Comment:
This should be inside the associated test method

Reply:
done.